### DISCUSSION TITLE
Highlighting text with QMarkdown 2.0.0 beta 4

### DISCUSSION THREAD

**Discussion by mickey-bimlive** [quasarframework/quasar-ui-qmarkdown#365] [General]
@hawkeye64
Hi Jeff, as I'm currently migrating our app to Quasar v2/Vue v3, I have upgraded QMarkdown to 2.0.0 beta 4 this week (still in test).

One glitch I'm seeing is that all highlighted text (format: "...this is highlighted in `==yellow==`") remains unformatted, means it is not highlighted in yellow any more. I'm not experiencing this on Quasar v1/QMarkdown 1.4.3. Is this a bug or due to some other reason? I already tried the prop "no-highlight='false'", but it has no effect on it.

Our existing markdown content happens to have a lot of text that got highlighted using `== ... ==`.
Regards, Michael

**Comment by hawkeye64** [comment]
To make QMarkdown not as heavy as it was, several plugins were removed. But, don't worry, you can easily add any of them back based on your needs. 

https://qmarkdown.netlify.app/developing/using-qmarkdown#Mark

**Comment by mickey-bimlive** [comment]
@hawkeye64  - thanks. So I'm trying to add this support for 'mark' which is now optional through a boot file as a global prop for q-markdown, as described in the docs. The boot file is:

```
import { useQMarkdownGlobalProps } from '@quasar/quasar-ui-qmarkdown'
import mark from 'markdown-it-mark' // https://www.npmjs.com/package/markdown-it-mark - installed via npm install markdown-it-mark --save
// defaults for QMarkdown
useQMarkdownGlobalProps({
  plugins: [mark]
})
```

However, `==marked text==` in my markdown content is still not processed after building with the above boot file (regular markdown content works fine though).

**Comment by mickey-bimlive** [comment]
It looks like that, for whatever reason, this extension 'mark' has changed the markdown format from `==marked text==` to `<mark>marked text</mark>`. Changed some of my markdown to this new format - now QMarkdown with the above global prop for 'mark' display it correctly.

**Reply by hawkeye64** [reply]
The `<mark>` tag is what the `markdown-it-mark` plugin does when it encounters a set of encapsulating `==` (<-- markdown)

**Comment by mickey-bimlive** [comment]
Not sure what your last comment means. In my case, only if I change my markdown from "==text==" to "text" enclosed by mark tags, the yellow highlighting works (through mark  installed as global property for QMarkdown). The original "==text==" is ignored in formatting.

**Comment by hawkeye64** [comment]
Try it locally, the same way as the example above, and see if that works for you.

**Comment by mickey-bimlive** [comment]
I've tried this meanwhile a couple of times, but still no success. I worked around the issue by changing all of my markdown content from the markdown format for marking (double equation symbols)  to native html mark tags, but still have no clue why the  old way of highlighting with the markdown-it-mark plugin globally enabled for q-markdown, as described above, does not work in my case. All other markdown works fine after my upgrade to Vue 3/Quasar 2/QMarkdown 2.0

**Comment by nucle** [comment]
Hi, did you solve it?

BR

**Comment by mickey-bimlive** [comment]
This is unresolved. Had to change all of my markdown from double equations symbols to html mark tags, to get highlighting of text.

**Comment by hawkeye64** [comment]
This works as documented.
